guides/source/configuring.md

```diff
@@ -112,6 +112,8 @@ numbers. It also filters out sensitive values of database columns when call `#in
 
 * `config.force_ssl` forces all requests to be served over HTTPS, and sets "https://" as the default protocol when generating URLs. Enforcement of HTTPS is handled by the `ActionDispatch::SSL` middleware, which can be configured via `config.ssl_options` - see its [documentation](https://api.rubyonrails.org/classes/ActionDispatch/SSL.html) for details.
 
+* `config.javascript_path` sets the path where your app's JavaScript lives relative to `/app`. The default is `javascript`, used by [webpacker](https://github.com/rails/webpacker). An app's configured `javascript_path` will be excluded from `autoload_paths`.
```

Comment:
```suggestion
* `config.javascript_path` sets the path where your app's JavaScript lives relative to `app` directory. The default is `javascript`, used by [webpacker](https://github.com/rails/webpacker). An app's configured `javascript_path` will be excluded from `autoload_paths`.
```

Reply:
Added "the" before `app`, but I agree this is better. Thanks!